webextensions/api/scripting.json

```diff
@@ -0,0 +1,351 @@
+                "firefox_android": {
+                  "version_added": "101"
+                },
+                "opera": {
+                  "version_added": "74"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              }
+            }
+          },
+          "world": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "88"
+                },
+                "edge": {
+                  "version_added": "88"
+                },
+                "firefox": {
+                  "version_added": "101",
+                  "notes": "Only supports <code>ISOLATED</code>, not <code>MAIN</code>."
+                },
+                "firefox_android": {
+                  "version_added": "101",
+                  "notes": "Only supports <code>ISOLATED</code>, not <code>MAIN</code>."
+                },
+                "opera": {
+                  "version_added": "74"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              }
+            }
+          }
+        },
+        "getRegisteredContentScripts": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/getRegisteredContentScripts",
+            "support": {
+              "chrome": {
+                "version_added": "88"
+              },
+              "edge": {
+                "version_added": "88"
+              },
+              "firefox": {
+                "version_added": "101"
+              },
+              "firefox_android": {
+                "version_added": "101"
+              },
+              "opera": {
+                "version_added": "74"
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            }
+          }
+        },
+        "insertCSS": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/insertCSS",
+            "support": {
+              "chrome": {
+                "version_added": "88"
+              },
+              "edge": {
+                "version_added": "88"
+              },
+              "firefox": {
+                "version_added": "101"
+              },
+              "firefox_android": {
+                "version_added": "101"
+              },
+              "opera": {
+                "version_added": "74"
+              },
+              "safari": {
+                "version_added": "15.4",
+                "notes": "Doesn't support <code>origin</code>."
```

Comment:
Could you add a new entry for "origin" instead of making it a comment?

(also in removeCSS)